packages/blade/src/components/Form/Selector/SelectorSupportText.tsx

```diff
@@ -1,27 +1,29 @@
-import { BaseText } from '~components/Typography/BaseText';
+import { Box } from '~components/Box';
+import { Text } from '~components/Typography/Text';
 import { getPlatformType } from '~utils';
 
 const SelectorSupportText = ({
   children,
   id,
+  isNegative,
 }: {
   children: React.ReactNode;
   id?: string;
+  isNegative?: boolean;
 }): React.ReactElement => {
   const isReactNative = getPlatformType() === 'react-native';
 
   return (
-    <BaseText
-      id={id}
-      as={isReactNative ? undefined : 'span'}
-      color="surface.text.muted.lowContrast"
-      fontSize={50}
-      lineHeight={50}
-      fontStyle="italic"
-      fontFamily="text"
-    >
-      {children}
-    </BaseText>
+    <Box id={id} display="contents">
```

Comment:
`display: "contents"` seems to have [partial support](https://caniuse.com/css-display-contents).  Would it cause any a11y issues, ([reference](https://hidde.blog/more-accessible-markup-with-display-contents/#heading-4))?

Reply:
> However, there is one caveat: none of the browsers that currently support display: contents expose elements that have the property to the accessibility tree with their original role.


This is problematic only when being used with an element that has an a11y role. In this case there is no a11y role for the box and it should be fine

Reply:
Cool, it looks like we also need to update the types to accept `contents.`

<img width="1123" alt="Screenshot 2024-01-09 at 4 47 06 PM" src="https://github.com/razorpay/blade/assets/46647141/50ef9bbe-ea78-4aa8-8312-8be40a33415c">

Reply:
Yup have done that too